jbpm-wb-integration/jbpm-wb-integration-api/src/main/java/org/jbpm/workbench/wi/workitems/model/ServiceTaskSummary.java

```diff
@@ -95,18 +102,33 @@ public Boolean getEnabled() {
     public void setEnabled(Boolean enabled) {
         this.enabled = enabled;
     }
-    
+
     public Set<String> getInstalledOn() {
         return installedOn;
     }
-    
+
     public void setInstalledOn(Set<String> installedOn) {
         this.installedOn = installedOn;
     }
 
+    public List<String> getParameters() {
+        return parameters;
+    }
+
+    public void setParameters(List<String> parameters) {
+        this.parameters = parameters;
+    }
+
+    public String getReferenceLink() {
+        return referenceLink;
+    }
+
+    public void setReferenceLink(String referenceLink) {
+        this.referenceLink = referenceLink;
+    }
+
     @Override
     public String toString() {
         return "ServiceTaskSummary [icon=" + icon + ", name=" + name + ", description=" + description + ", enabled=" + enabled + "]";
```

Comment:
@mswiderski can you add these new attributes to the toString, if reasonable.

Reply:
I left it with this base set of information on purpose to avoid too much in toString

Reply:
np